main.py

```diff
@@ -2,6 +2,47 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    # create
+    genres_names_to_create = ["Western", "Action", "Dramma"]
+    for genre_name in genres_names_to_create:
+        Genre.objects.create(
+            name=genre_name
+        )
+
+    input_actors_fullnames_to_create = [
+        "George Klooney", "Kianu Reaves", "Scarlett Keegan",
+        "Will Smith", "Jaden Smith", "Scarlett Johansson"
+    ]
+    actors_names_to_create = (
+        (actor_name.split()[0], actor_name.split()[1])
+        for actor_name
+        in input_actors_fullnames_to_create
+    )
```

Comment:
I left the comment about it . You can try it 
For example --- >
```
actors_data = [
    ( first_name, last_name),
    (... , ... )
]
```
Then --- >

`for first_name , last_name in actors_data:`